.github/workflows/eslint.yml

```diff
@@ -0,0 +1,30 @@
+name: Lint # name of the action (displayed in the github interface)
+
+on: # event list
+pull_request: # on a pull request to each of these branches
+branches:
+  - development
+  - staging
+  - production
```

Comment:
Looks good, you may not need to specify branches here, you can just specify for it to run on pull_requests.
`on: [pull_request, workflow_dispatch]` for this on line 3, we may not have any of the branches you listed above

you can check more information https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows